server/views/pages/confirmation.njk

```diff
@@ -0,0 +1,174 @@
+{% extends "layout.njk" %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+
+{% set pageHeaderTitle = "Booking confirmed" %}
+{% set pageTitle %}
+{{ applicationName }} - {{ pageHeaderTitle }}
+{% endset %}
+
+{% block content %}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        <h1 class="govuk-heading-l"></h1>
+        {{ govukPanel({
+            titleText: pageHeaderTitle,
+            html: 'The reference number is<br><strong class="test-booking-reference">' + visitId + "</strong>"
+        }) }}
+    </div>
+</div>
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        {% include "partials/errorSummary.njk" %}
+        <h2 class="govuk-heading-m govuk-!-margin-top-6">What to do next</h2>
+        <p>Let the caller know:</p>
+        <ul class="govuk-body">
+            <li>the main contact will receive a text message with the reference number and visit details</li>
+            <li>they will need the reference number to <a href="#">cancel or amend the booking</a></li>
+            <li>changes to the booking can be made up to 2 working days before the visit</li>
+            <li>who can <a href="#">get help with the cost of visits</a></li>
+        </ul>
+        <p>To prepare for the visit day, tell them:</p>
+        <ul class="govuk-body">
+            <li>what time to arrive for the visit</li>
+            <li>all adult visitors will need to bring <a href="#">acceptable forms of ID</a></li>
+            <li>any COVID-19 requirements (such as face coverings and lateral flow tests)</li>
+            <li>any prison specific rules (such as clothing guidelines)</li>
+        </ul>
+        {{ govukButton({
+            classes: "govuk-!-margin-top-3, govuk-!-margin-bottom-6",
+            text: "Manage prison visits"
+        }) }}
+    </div>
+</div>
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        <h2 class="govuk-heading-m">Visit details</h2>
+        {{ govukSummaryList({
```

Comment:
Does this end up pretty similar to the content on check your answers page - could perhaps have it as an include template?

Reply:
Irritatingly it's not the same, a few bits are, maybe I can pull out those sections which are the same.

Reply:
I'll do this after I've merged this.